<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QWidget" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_a">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_b">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_c">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>450</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_a">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>160</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_b">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>160</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_c">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_d">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>129</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reset">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>297</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>重置(R)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>297</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出(X)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_equal">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>240</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>lineEdit_a</zorder>
  <zorder>lineEdit_b</zorder>
  <zorder>lineEdit_c</zorder>
  <zorder>radioButton_a</zorder>
  <zorder>radioButton_b</zorder>
  <zorder>radioButton_c</zorder>
  <zorder>radioButton_d</zorder>
  <zorder>pushButton_reset</zorder>
  <zorder>pushButton_exit</zorder>
  <zorder>pushButton_equal</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
